fluidBuild.config.cjs

```diff
@@ -46,7 +46,7 @@ module.exports = {
 		"build:genver": [],
 		"typetests:gen": ["^tsc", "build:genver"], // we may reexport type from dependent packages, needs to build them first.
 		"tsc": tscDependsOn,
-		"build:esnext": tscDependsOn,
+		"build:esnext": [...tscDependsOn, "^build:esnext"],
```

Comment:
Now that we've started being explicit about CJS and ESM entrypoints, the ESM build should use the ESM output from dependencies (in the past it may have used the CJS output). Thus, we now need the upstream ESM builds to complete to ensure we can build ESM locally.